docs/docs/contributing/developer-guide.md

```diff
@@ -33,8 +33,34 @@ As of now, we only support integration tests who interact directly with Aiven. T
 an [Aiven account](https://console.aiven.io/signup?utm_source=github&utm_medium=organic&utm_campaign=k8s-operator&utm_content=signup)
 and an [Aiven authentication code](https://help.aiven.io/en/articles/2059201-authentication-tokens).
 
+### Prerequisites
+
+Please have installed first:
+
+- docker/podman
+- [kind](https://kind.sigs.k8s.io/)
+- [helm](https://helm.sh/)
+- [Aiven CLI](https://docs.aiven.io/docs/tools/cli) (make sure you have logged in)
+- [jq](https://stedolan.github.io/jq/)
+- [kcat](https://github.com/edenhill/kcat)
+- base64 (note: MACOS version doesn't support `-w0` flag, some tests may not work properly)
+
+Create a cluster:
+
+```shell
+ kind create cluster --image kindest/node:v1.24.0 --wait 5m
+```
+
+Setup everything:
+
+```shell
+AIVEN_TOKEN="<your-token>" AIVEN_PROJECT_NAME="<your-project-name>" make e2e-setup-kind
```

Comment:
I have a problem running this command on my box:
```
✗ make e2e-setup-kind 
K :=  some string  some string  some string  some string  some string  some string  some string
bash: line 1: K: command not found
make: *** [Makefile:308: e2e-setup-kind] Error 127
```

Reply:
Thanks. I have removed prerequisites check. That might be the difference between mac and linux. Please try again.